samples/fabric/fabric-cli/src/helpers/helpers.ts

```diff
@@ -832,19 +832,23 @@ const interopHelper = async (
       keyCert,
       false,
       options['relay-tls'] === 'true',
-      relayTlsCAFiles
+      relayTlsCAFiles,
+      options['e2e-confidentiality'] === 'true' ? true : false
```

Comment:
do we need tertiary operator here? Isn't the return value itself is boolean?

Reply:
Done